bridgeservice/bridge.go

```diff
@@ -85,6 +86,14 @@ func New(
 	meter := otel.Meter(meterName)
 	cfg.Logger.Infof("starting bridge service (network id=%d)", cfg.NetworkID)
 
+	ginMode := os.Getenv("GIN_MODE")
```

Comment:
@Stefan-Ethernal where are we setting this var?

Reply:
Currently nowhere, and it fallbacks to `release` mode. I'm unsure what is the best approach for this should we create two targets in the `Makefile`:
- `build-debug` - builds a debug Docker image
- `build-release` - builds a release Docker image

Reply:
yes but then this will change the workflow which is already set for others
maybe we can set this to `release` mode in makefile and put a comment that if someone wants to build with a diff mode, modify this 
something like this

Reply:
Fixed https://github.com/agglayer/aggkit/pull/515/commits/adc650512a35affbbd91161dd851c09a0e405804